contracts/Strategy.sol

```diff
@@ -747,12 +754,20 @@ contract Strategy is BaseStrategy, DydxFlashloanBase, ICallee {
         }
     }
 
+    //emergency function that we can use to deleverage manually if something is broken
+    function manualDeleverage(uint256 amount) external management{
+        require(cToken.redeemUnderlying(amount) ==0, "failed redeem");
+        cToken.repayBorrow(amount);
```

Comment:
If we call `manualReleaseWant()` we cannot later call manualDeleverage since the require may fail?

Main question if `ctoken.repayBorrow` needs its own method to separate that step like `manualReleaseWant` only redeems?